app/components/layer-file-settings/component.js

```diff
@@ -1,4 +1,15 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
+	file_object: null,
+	actions: {
+       fileDetail(file) {
+       		this.set('layer.settings.properties.downloadLink' , file._internalModel.__data.links.download)
+        },
+
+        nodeDetail(node) {
+            this.transitionToRoute('nodes.detail', node);
```

Comment:
Do you need to do this? It doesn't seem pertinent for us, we will not go into the node. You can just leave empty.